pkg/sdk/parameters_impl.go

```diff
@@ -199,85 +173,123 @@ func GetSessionParametersUnsetFrom(params map[string]any) (*SessionParametersUns
 }
 
 func (sessionParametersUnset *SessionParametersUnset) setParam(parameter SessionParameter) error {
```

Comment:
Nit: I know this is not a part of this PR, but shouldn't the name be `unsetParam`?

Reply:
hmmm, I think both are valid: it's setting the given param on the unset object, so this is setting a param after all :)